downstream/modules/automation-hub/proc-create-synclist.adoc

```diff
@@ -2,34 +2,32 @@
 // obtaining-token/master.adoc
 [id="proc-create-synclist"]
 
-= Creating a synclist of Red Hat Certified collections
+= Creating a synclist of Red Hat {CertifiedName}
 
-You can create a synclist of curated Red Hat Certified collections in {HubName} on console.redhat.com. 
-Your synclist repository is located under menu:Automation Hub[Repo Management], which is updated whenever you choose to manage content within Red Hat Certified collections.
+You can create a synclist of curated {CertifiedCon} in {HubNameMain} on {Console}. 
+Your synclist repository is located under menu:Automation Hub[Repo Management], which is updated whenever you choose to manage content within {CertifiedName}.
 
-By default, all Red Hat Certified collections are included in your initial organization synclist.
+All {CertifiedName} are included by default in your initial organization synclist.
 
 .Prerequisites
 
 * You have a valid {PlatformNameShort} subscription.
-* You have Organization Administrator permissions for console.redhat.com.
+* You have Organization Administrator permissions for {Console}.
```

Comment:
Is the Organization Administrator role still valid.

Reply:
Ignoring this as it seems to apply to keycloak, which isn't relevant in this case.